test/widgets/home_test.dart

```diff
@@ -441,5 +443,22 @@ void main () {
       // No additional wait for loadPerAccount.
       checkOnHomePage(tester, expectedAccount: eg.selfAccount);
     });
+
+    testWidgets('logging out', (tester) async {
+      // Regression test for: https://github.com/zulip/zulip-flutter/issues/1219
+      addTearDown(testBinding.reset);
+      await testBinding.globalStore.add(eg.selfAccount, eg.initialSnapshot());
+      await tester.pumpWidget(const ZulipApp());
+      await tester.pump(Duration.zero); // wait for the loading page
+      await tester.pump(loadPerAccountDuration);
```

Comment:
What's the role of `loadPerAccountDuration` here? It doesn't look like anything else in the test refers to it.

Reply:
Yeah it is not needed.